base/dotenv.default.mustache

```diff
@@ -0,0 +1,3 @@
+PORT=
+RETHINK_HOST=
+RETHINK_NAME=
```

Comment:
I didn't use it because it seemed to exist for the purpose of simply using as a template to create a `.env`, so I was concerned that users might remove `example.env` after creating a `.env`, and not realising it was used for checking env vars exist.

But I like the idea as it's one less file.

Actually, perhaps we do use it, and it's also a free way to provide users a notification to create an `.env` file?

... actually actually, why do we create `example.env` and not just `.env`?